specification/compute/resource-manager/Microsoft.Compute/ComputeRP/stable/2023-09-01/examples/sshPublicKeyExamples/SshPublicKey_GenerateKeyPairEd25519.json

```diff
@@ -0,0 +1,18 @@
+{
+  "parameters": {
```

Comment:
please update this file name to:  SshPublicKey_GenerateKeyPair_WithEncryption.json